### PR TITLE
docs(recommendations): grammar fixes

### PR DESCRIPTION
**Pull request by Fdawgs**
#### Checklist

- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by jsumners — APPROVED** [review]
Looks good to me once CI passes.

**Review by mcollina — APPROVED** [review]
lgtm

File: docs/Guides/Recommendations.md [hunk 1]
```diff
@@ -305,32 +305,32 @@ readinessProbe:
...
 
-You can use such tools as [k6](https://github.com/grafana/k6) 
-or [autocannon](https://github.com/mcollina/autocannon) for conducting
-the necessary performance tests.
+Tools such as [k6](https://github.com/grafana/k6) 
+or [autocannon](https://github.com/mcollina/autocannon) can be used for
+conducting the necessary performance tests.
 
-That said, you may also consider the following as a rule of a thumb:
+That said, you may also consider the following as a rule of thumb:
```
Comment (Uzlopak) [inline docs/Guides/Recommendations.md:318]: @Fdawgs 
Why not rule of two thumbs? Every human has two of them?

File: docs/Guides/Recommendations.md [hunk 2]
```diff
@@ -305,32 +305,32 @@ readinessProbe:
...
 
-* In order to optimize for throughput (handling the largest possible amount of
-requests per second per vCPU available), consider using smaller amount of vCPUs
+* To optimize for throughput (handling the largest possible amount of
+requests per second per vCPU available), consider using a smaller amount of vCPUs
 per app instance. It is totally fine to run Node.js application with 1 vCPU.
 
-* You may experiment with even smaller amount of vCPU, which may provide even 
-better throughput in certain use-cases. There are reports of e. g. API gateway
+* You may experiment with an even smaller amount of vCPU, which may provide even
```
Comment (Uzlopak) [inline docs/Guides/Recommendations.md:330]:
```suggestion
* You may experiment with an even smaller amount of vCPU, which may provide 
```

File: docs/Guides/Recommendations.md [hunk 3]
```diff
@@ -305,32 +305,32 @@ readinessProbe:
...
-* In order to optimize for throughput (handling the largest possible amount of
-requests per second per vCPU available), consider using smaller amount of vCPUs
+* To optimize for throughput (handling the largest possible amount of
+requests per second per vCPU available), consider using a smaller amount of vCPUs
 per app instance. It is totally fine to run Node.js application with 1 vCPU.
 
-* You may experiment with even smaller amount of vCPU, which may provide even 
-better throughput in certain use-cases. There are reports of e. g. API gateway
+* You may experiment with an even smaller amount of vCPU, which may provide even 
+better throughput in certain use-cases. There are reports of API gateway
```
Comment (Uzlopak) [inline docs/Guides/Recommendations.md:331]:
```suggestion
even better throughput in certain use-cases. There are reports of API gateway
```

File: docs/Guides/Recommendations.md (hunk 1 above)
Comment (Fdawgs) [reply docs/Guides/Recommendations.md:318]: 🤦

**Pull request merged**